hi-ml-histopathology/src/histopathology/configs/classification/BaseMIL.py

```diff
@@ -5,30 +5,32 @@
 import os
 import torch
 import param
+from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
 
 from torch import nn
 from pathlib import Path
 from monai.transforms import Compose
 from torchvision.models import resnet18
```

Comment:
can we import resnet18 and resnet50 here, so we don't need to import them separately?